### PR TITLE
✨ Add payload event

### PR DESCRIPTION
**Pull request by ghost**
### Changes

-   `adds`: A `payload` event, which forwards any gateway messages.

 ### Check off the following

-   [x] I have tested my changes with the current requirements
-   [x] My Code follows the pep8 code style.

### CODE REVIEW

File: pincer/middleware/payload.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+# CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+"""sent when basically anything is received from gateway"""
```
Comment (Sigmanificient) [inline pincer/middleware/payload.py:25]: can you update to the compressed license please?

File: pincer/core/gateway.py [hunk 2]
```diff
@@ -107,7 +110,7 @@ async def handle_reconnect(_, payload: GatewayDispatch):
             7: handle_reconnect,
             9: handle_reconnect,
             10: identify_and_handle_hello,
-            11: Heartbeat.handle_heartbeat
+            11: Heartbeat.handle_heartbeat,
```
Comment (Sigmanificient) [inline pincer/core/gateway.py:113]: please remove that `,`

File: pincer/core/gateway.py [hunk 3]
```diff
@@ -22,6 +22,9 @@
     _InternalPerformReconnectError, DisallowedIntentsError
 )
 from ..objects import Intents
+from ..middleware.payload import (
```
Comment (Sigmanificient) [inline pincer/core/gateway.py:25]: No need for paranthesis here, as
```py
from ..middleware.payload import payload_middleware
```
is `< 80` charaters

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

File: pincer/middleware/payload.py (hunk 1 above)
Comment (Sly-Little-Fox) [reply pincer/middleware/payload.py]: Already noticed, updated

**Comment by Sly-Little-Fox** [comment]
Why does GH think I merged it??? Recreating.

**Comment by Sigmanificient** [comment]
> Why does GH think I merged it??? Recreating.

I think GitHub auto-detect your push in the `Pincer/main` as a merge confirm xD

**Comment by Sly-Little-Fox** [comment]
> > Why does GH think I merged it??? Recreating.
> 
> I think GitHub auto-detect your push in the `Pincer/main` as a merge confirm xD

Probably

**Pull request merged**